packages/stark-testing/helpers.js

```diff
@@ -0,0 +1,11 @@
+const path = require("path");
+
+/**
+ * Helper functions.
+ */
+// const ROOT = path.resolve(__dirname, '..');
+const _root = path.resolve(process.cwd(), "."); // project root folder
+
+const root = path.join.bind(path, _root);
```

Comment:
Well, the idea is to make Stark as flexible as possible, so normally all the testing tooling is just a decision we made in Stark and should not have any impact in the final users of Stark (install our deps in their project) if they decide use another tooling.

So, we should keep this package as independent as possible, because again, Stark is really flexible and the stark-build might not be the package selected by the user to build their app...

Reply:
In any case, we can review this later once we have a better view on how all our packages will be used in a client app.

Reply:
Ok for me